gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -857,7 +857,6 @@ class VeloxTestSettings extends BackendTestSettings {
     // decimal failed ut
     .exclude("SPARK-34212 Parquet should read decimals correctly")
     // Timestamp is read as INT96.
```

Comment:
This message was added when int64 type was unsupported for timestamp reader. Could you update the message to illustrate why below tests are still disabled?

> .exclude("Migration from INT96 to TIMESTAMP_MICROS timestamp type")
    .exclude("SPARK-10365 timestamp written and read as INT64 - TIMESTAMP_MICROS")

Reply:
These two unit tests failed because int64 timestamp read is not supported.